bitrise.yml

```diff
@@ -0,0 +1,184 @@
+format_version: 4
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # An example secret param, define it (A_SECRET_PARAM) in .bitrise.secrets.yml
+  - A_SECRET_PARAM: $A_SECRET_PARAM
+  # If you want to share this step into a StepLib
+  - BITRISE_STEP_ID: create-zip
+  - BITRISE_STEP_VERSION: "0.0.1"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/birmachera/bitrise-step-create-zip
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+
+
+workflows:
+  godeps-update:
```

Comment:
use dep tool to manage go dependencies: https://github.com/bitrise-io/steps-xcode-archive/blob/master/bitrise.yml#L186